webviews/preview_compiled/App.tsx

```
@@ -0,0 +1,94 @@
+                <Loader2 className="w-5 h-5 animate-spin" />
+                <span>Compiling Dataform...</span>
+            </div>
+        )}
+
+
+
+        {state.errorMessage && !state.recompiling && (
+            <div className="bg-red-900/20 border-l-4 border-red-600 p-4 mb-4 rounded-r shadow-sm">
+                <div className="flex items-start">
+                    <AlertCircle className="w-5 h-5 text-red-500 mt-0.5 mr-2 flex-shrink-0" />
+                    <div className="text-red-200 text-sm overflow-auto" dangerouslySetInnerHTML={{__html: state.errorMessage}} />
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)